packages/components/src/data-model/constants/error-codes.js

```diff
@@ -1,9 +1,12 @@
+/*
+All possible error and warning codes that can be returned from the GraphQL API.
+ */
 /** @enum {string} */
-const WordQueryErrorCodes = {
+const ErrorCodes = {
   TUFTS_ERROR: 'TUFTS_ERROR',
   TREEBANK_ERROR: 'TREEBANK_ERROR',
   LEXICONS_ERROR: 'LEXICONS_ERROR',
   DISAMBIGUATION_ERROR: 'DISAMBIGUATION_ERROR',
   SHORT_DEFS_ERROR: 'SHORT_DEFS_ERROR'
 }
-export default WordQueryErrorCodes
+export default ErrorCodes
```

Comment:
I think this list of error codes can belong not to WordQuery only but can be shared across other queries. Hence the renaming.